ledger-service/http-json/src/main/scala/com/digitalasset/http/util/Commands.scala

```diff
@@ -3,7 +3,9 @@
 
 package com.daml.http.util
 
+import com.daml.http.domain
 import com.daml.ledger.api.refinements.{ApiTypes => lar}
+import com.daml.ledger.api.v1.commands.Commands.DeduplicationPeriod
 import com.daml.ledger.api.{v1 => lav1}
```

Comment:
Just a small cleanup, take it or leave it.

```suggestion
import com.daml.ledger.api.{v1 => lav1}
import lav1.commands.Commands.DeduplicationPeriod
```